src/test/java/teammates/test/cases/action/BaseActionTest.java

```diff
@@ -136,6 +136,23 @@ protected ImageResult getImageResult(Action a) {
         };
     }
 
+    protected String[] createInvalidParamsForProfileWithScriptInjection() {
```

Comment:
This method should not belong to `BaseActionTest`. Instead, put it to the place where the method is used.